README.md

```diff
@@ -29,44 +29,48 @@ With the repo cloned, proceed to checkout to the dev branch, `git checkout dev`.
 
-```
+``` script
 docker-compose down
 ```
 
 which kills your containers but keeps your iamges. Be sure to use `docker-compose --help` for any additional help or other options.
 
-If the code has been changed, you need to rebuild your image. Due to the fact that we use a docker-compose.yml, please look at the service name you are looking to rebuild and run:
+If the code has been changed, you need not rebuild using `docker-compose up -d --build` for backend or mongo but as of this moment and vue file changes will need to run the command.
 
-```
-docker-compose build your-services-name
+To access the website, the static files will be served on `0.0.0.0:8080` (`[::]:8080`) while the backend APIs are served on `0.0.0.0:1323` (`[::]:1323`). Make sure when you are making calls from the frontend to the backend in development stage, you use the suffix of the api call and not call with the domain e.g
+
+``` javascript
+/api/v1/sponsors
 ```
 
-And then proceed with previous steps to get start local deployment.
+as opposed to
 
-The server will start on `0.0.0.0:1323` (`[::]:1323`) which serves both the frontend and API simultaneously.
+``` javascript
+https://localhost:1323/api/v1/sponsors
+```
 
 ## Running the tests
 
-The project uses Github Actions for continuous integration and automated testing.
+The project uses Github Actions for continuous integration and automated testing. Testing will always be written at the beginning of each sprint and run every time a push is detected on your feat/fix/hotfix branch or a merge/push to dev.
 
 ### Input Validation
-To validate test, we are utilising a feature of the echo web framework that allows us to couple a validator package to validate structs that contain user inputs from requests. The package is golang's [package validator](https://pkg.go.dev/gopkg.in/go-playground/validator.v9?tab=doc#pkg-index). Everytime validation needs to occur for inserting into a database please use `echo-context.Validator(&struct)` to validate and handle any errors accordingly.
+To validate structs, we are utilising a feature of the echo web framework that allows us to couple a validator package to validate structs that contain user inputs from requests. The package is golang's [package validator](https://pkg.go.dev/gopkg.in/go-playground/validator.v9?tab=doc#pkg-index). Everytime validation needs to occur for inserting into a database please use `echo-context.Validator(&struct)` to validate and handle any errors accordingly.
+
+### Unit and Integration Testing
+These tests are written in golang's standard testing package and are written in the same package as the file that they are testing. To have these test run, please ensure docker is running and your containers for development are running. Then to run the actual test, go to the directory that contains the go files and run `go test`. This will run all test in child directories. 
```

Comment:
You should include how the test files should be named with the *_test.go suffix. For a given program main.go, the test file must be named main_test.go

Reply:
Did a brief one liner and directed them to the website for further info.